src/components/latestPosts/latestPosts.tsx

```diff
@@ -0,0 +1,36 @@
+import clsx from "clsx";
+import React from "react";
+import styles from "./styles.module.css";
+
+export default function LatestPosts({ recentPosts }) {
+  return (
+    <div className={clsx("container", styles.wrap)}>
+      <h2>Latest posts and feature explainers</h2>
```

Comment:
```suggestion
      <h2>Latest posts</h2>
```
Just for simplicity.